packages/devtools_app/lib/src/config_specific/flutter/import_export/_export_desktop.dart

```diff
@@ -15,7 +15,8 @@ class ExportControllerDesktop extends ExportController {
   static final _fs = FileIO();
 
   @override
-  void downloadFile(String filename, String contents) {
+  String downloadFile(String contents) {
```

Comment:
returning the filename from a method called downloadFile is surprising to me. Can you leave the method as void?

Reply:
we show a notification "Successfully exported $exportedFile to ~/Downloads directory", so we need the exported file name